src/prelude.rs

```diff
@@ -16,9 +16,3 @@ pub use crate::rcc::RccExt as _stm32f3xx_hal_rcc_RccExt;
 pub use crate::syscfg::SysCfgExt as _stm32f3xx_hal_syscfg_SysCfgExt;
 pub use crate::time::duration::Extensions as _stm32f3xx_hal_time_time_Extensions;
 pub use crate::time::rate::Extensions as _stm32f3xx_hal_time_rate_Extensions;
-pub use crate::{
```

Comment:
This is an important breaking change, which should be commented especially in the Changelog. 

Might also be a good idea to put this in a single commit :)